R/transpose_pretty.R

```diff
@@ -176,10 +176,12 @@ widen_tabulation <- function(y, cod, st, pretty = TRUE, digits = 1) {
   y <- tidyr::spread(y, !!TMP, !!VALUE)                                       # 6
 
   # 7
-  levels_in_order <- glue::glue("^{l} (n|deff|proportion|ci)")
-  col_arrangement <- lapply(levels_in_order, grep, names(y))
-  col_arrangement <- unlist(col_arrangement, use.names = FALSE)
-  y               <- y[c(1, col_arrangement)]
+  # guarding against common situation of having a stratifier that is 45+, changing to 45[+]
```

Comment:
It’s not that it changes the output in any way, it’s just modifying the search pattern. 

If you do grepl(“45+”, “45+“), it won’t match because it’s looking for 45 followed by one or more 5s. This makes the pattern grepl(“45[+]”, “45+”), which accurately matches the +.